gobbler/views/navBar.html

```diff
@@ -1,7 +1,7 @@
 <template name='navBar'>
-  <div class='nav-bar'>
```

Comment:
I noticed you took off the line with the class declaration here, but are still referencing it in the styling sheet. Is it classes elsewhere?